pynitrokey/cli/nethsm.py

```diff
@@ -1416,6 +1416,13 @@ def restore(
             )
 
     with connect(ctx, require_auth=False) as nethsm:
+        state = nethsm.get_state()
+        if state == State.UNPROVISIONED:
+            require_auth = False
+        elif state == State.OPERATIONAL:
+            require_auth = True
```

Comment:
what happens in the `else` case?  maybe just check for one of the two states, or directly raise an exception

Reply:
I change this to have a default to `require_auth = False`. Only the state `Operational` will change this. Otherwise the error is handled by the SDK, with error message from the NetHSM.